justfile

```diff
@@ -81,3 +81,7 @@ format:
 # start tmuxinator with a dev federation setup
 tmuxinator:
   ./scripts/tmuxinator.sh
+
+# exit tmuxinator session
+exit-tmuxinator:
+  tmux kill-session -t fedimint-dev
```

Comment:
@okjodom I wonder if also adding a new cmd to restart (using the existing script) would also be helpful, wdyt ? 😁

Reply:
You can just propose that alias. Would be useful!